### PR TITLE
Update Foreman Gem and NPM dependencies

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -216,8 +214,6 @@ BuildRequires: %{?scl_prefix}rubygem(sprockets) >= 4.0
 BuildRequires: %{?scl_prefix}rubygem(sprockets) < 5.0
 BuildRequires: %{?scl_prefix}rubygem(sprockets-rails) >= 3.0
 BuildRequires: %{?scl_prefix}rubygem(sprockets-rails) < 4.0
-BuildRequires: %{?scl_prefix}rubygem(record_tag_helper) >= 1.0
-BuildRequires: %{?scl_prefix}rubygem(record_tag_helper) < 2.0
```
Comment (ekohl) [inline packages/foreman/foreman/foreman.spec:220]: We shouldn't forget to drop this, preferably before branching. @zjhuntin @pcreech @Odilhao.

Comment (evgeni) [reply packages/foreman/foreman/foreman.spec:220]: ?

Comment (ekohl) [reply packages/foreman/foreman/foreman.spec:220]: The `rubygem-record_tag_helper` is now no longer used so we can retire the package.

Comment (evgeni) [reply packages/foreman/foreman/foreman.spec:220]: ah! this = the package, not this = this hunk. gotcha.

Comment (ekohl) [reply packages/foreman/foreman/foreman.spec:220]: Yes, early morning reviews aren't the best. I opened https://github.com/theforeman/foreman-packaging/pull/7513 which should do this.

**Pull request merged**